e2e/app.spec.ts

```diff
@@ -2,6 +2,8 @@ import { test, expect } from '@playwright/test';
 import { initAsUsualUser } from './helpers/e2e_auth';
 import { apiUrl } from './helpers/e2e_http';
 
+test.describe.configure({ mode: 'parallel' });
```

Comment:
Why there? Is it specific to this file?  
it is different from what is inside the config file `/playwright.config.ts`?

Reply:
Fixed